src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_PROPERTY_SIZE = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUMBER_OF_FIGURES)) {
+            case 0:
+                return getRandomRectangle();
+            case 1:
+                return getRandomRightTriangle();
+            case 2:
+                return getRandomSquare();
+            case 3:
+                return getRandomIsoscelesTrapezoid();
+            case 4:
+                return getRandomCircle();
+            default:
+                throw new NullPointerException();
```

Comment:
You don't need to throw `NullPointerException`. Better to return `getRandomCircle` instead of `case 4:` or call  `getDefaultFigure`